wl-sim/main/include/WL_Flash.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __WL_FLASH_H__
```

Comment:
#pragma once does the same job and you don't need to care about the symbols creations + have the code few #ifdefs shorter

Reply:
switched to using `#pragma once`